src/fileActions.jsx

```diff
@@ -563,3 +526,137 @@ const setDirectoryName = (val, setName, setNameError, setErrorMessage) => {
+            { type: "divider" },
+            {
+                id: "create-item",
+                title: _("Create directory"),
+                onClick: () => Dialogs.show(<CreateDirectoryModal currentPath={currentPath} />),
+            },
+            {
+                id: "create-link",
+                title: _("Create link"),
+                onClick: () => createLink(currentPath, files, selected || {})
+            },
+            { type: "divider" },
+            {
+                id: "edit-permissions",
+                title: _("Edit permissions"),
+                onClick: () => editPermissions(Dialogs, selected[0], path)
+            }
+        );
+    } else if (selected.length === 1) {
+        menuItems.push(
+            {
+                id: "copy-item",
+                title: _("Copy"),
+                // HACK: maybe it is the same as below?
+                onClick: () => setClipboard([currentPath + selected[0].name]),
+            },
+            ...(selected[0].type === "dir")
+                ? [
+                    {
+                        id: "paste-into-directory",
+                        title: _("Paste into directory"),
+                        onClick: () => spawnPaste(clipboard, [currentPath + selected[0].name], false, addAlert),
+                        isDisabled: clipboard === undefined
+                    }
+                ]
+                : [],
+            { type: "divider" },
+            {
+                id: "edit-permissions",
+                title: _("Edit permissions"),
+                onClick: () => editPermissions(Dialogs, selected[0], path)
+            },
+            {
+                id: "rename-item",
+                title: _("Rename"),
+                onClick: () => {
+                    Dialogs.show(
+                        <RenameItemModal
+                          path={path}
+                          selected={selected[0]}
+                        />
+                    );
+                },
+            },
+            { type: "divider" },
+            {
+                id: "create-link",
+                title: _("Create link"),
+                onClick: () => createLink(currentPath, files, selected[0]),
+            },
+            { type: "divider" },
+            {
+                id: "delete-item",
+                title: _("Delete"),
+                className: "pf-m-danger",
+                onClick: () => {
+                    Dialogs.show(
+                        <ConfirmDeletionDialog
+                          selected={selected} path={currentPath}
+                          setSelected={setSelected}
+                        />
+                    );
+                }
+            },
+        );
+    } else if (selected.length > 1) {
+        menuItems.push(
+            {
+                id: "copy-item",
+                title: _("Copy"),
+                onClick: () => setClipboard(selected.map(s => path.join("/") + "/" + s.name)),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-307-20240307-084539-34707c5b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)